src/LineChart/LineChart.xml

```diff
@@ -199,6 +199,11 @@
             <category>Appearance</category>
             <description/>
         </property>
+        <property key="refreshInterval" type="integer" defaultValue="0">
+            <caption>Refresh interval (ms)</caption>
+            <category>Appearance</category>
```

Comment:
`Refresh` is not a real logical property of Appearance. 
Please move `refreshInterval` from all charts into the category `Data source`

Reply:
The Data source tab is per series and yet this property is for the whole chart. should it be under the tab for Chart Properties then?